source/accessibility/index.html.md.erb

```diff
@@ -47,21 +49,30 @@ The Equality and Human Rights Commission (EHRC) is responsible for enforcing the
 
 ##### Compliance status
 
-This website is fully compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard.
+This website is partially compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard.
 
 ### Preparation of this accessibility statement
 
-This statement was prepared on 2 September 2020. It was last reviewed on 29 March 2021.
+This statement was prepared on 2 September 2020. It was last reviewed on 18 June 2021.
 
-This website was last tested in March 2021.
+This website was last tested in April 2021.
```

Comment:
Technically this is incorrect as this particular website (the TDT docs) hasn't been re-tested. So the date probably needs to stay as it was.